frontend/src/components/Users/InlineTextEdit.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-edit-dialog
+    :return-value.sync="item.notes"
+    lazy
+    large
+    @save="$listeners.save(item, editPath, item.notes)"
+    @cancel="$listeners.cancel"
```

Comment:
Why `listeners` is here?

Reply:
The save and cancel functions are passed as events.

Reply:
I mean, why it is called “listeners” in this context? I don't understand this.